resources/views/components/input/select/region.blade.php

```diff
@@ -10,6 +10,7 @@
         <template v-if="data && data.country.available_regions">
             <x-rapidez::input.select
                 {{ $attributes }}
+                v-if="'{{ Rapidez::config('general/region/state_required') }}'.split(',').includes({{ $country }}) || {{ Rapidez::config('general/region/display_all', '0') }}"
```

Comment:
```suggestion
                v-if="{{ Rapidez::config('general/region/display_all', '0') }} || '{{ Rapidez::config('general/region/state_required') }}'.split(',').includes({{ $country }})"
```
Putting the simpler check first may allow us to skip the more complex second check